sdk/spring/azure-spring-data-cosmos/src/samples/java/com/azure/spring/data/cosmos/AppConfiguration.java

```diff
@@ -49,6 +54,19 @@ public class AppConfiguration extends AbstractCosmosConfiguration {
     @Value("${azure.cosmos.responseContinuationTokenLimitInKb}")
     private int responseContinuationTokenLimitInKb;
 
+    @Value("${cosmos.diagnosticsThresholds.pointOperationLatencyThreshold}")
```

Comment:
Please add the Dimension (ms or S) in the suffix if teh name - similar like TokenLimitInKb above. My preference would be ms - cx definitely often want to use end-to-end operation timeouts more granular than seconds.

Reply:
Done.